test/utils_shared/mock_utils.py

```diff
@@ -35,58 +45,19 @@ def get_client_mocks(config, config_path, *to_be_mocked):
     Returns a dict of the class to the the class's mock or implementation as specified in
```

Comment:
`s/the the/the/`

Reply:
fixed, actually had this fixed on a different branch and forgot to fix it here